lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -90,36 +158,43 @@ def create(self, subsection, course_structure, course):
         """
         Returns the SubsectionGrade object for the student and subsection.
         """
+        self._prefetch_scores(course_structure, course)
         return (
-            self._get_saved_grade(subsection, course) or
+            self._get_saved_grade(subsection, course_structure, course) or
             self._compute_and_update_grade(subsection, course_structure, course)
         )
 
     def _compute_and_update_grade(self, subsection, course_structure, course):
```

Comment:
Should this be `_compute_and_save_grade` instead - since the grade didn't exist and should now be saved?